client/dist/index.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
   <div id="workspaces"></div>
-  <script src="https://workspaces-service.s3-us-west-1.amazonaws.com/dist/workspaces.js"></script>
+  <script src="https://workspaces-service.s3-us-west-1.amazonaws.com/dist/workspaces.js.br"></script>
```

Comment:
This is definitely one of the best compression algorithms and it is perfect for this project, however, keep in mind it's not compatible with every browser: https://caniuse.com/brotli